nox/__main__.py

```diff
@@ -216,6 +218,10 @@ def main():
         "--report", help="Output a report of all sessions to the given filename."
     )
 
+    secondary.add_argument(
+        "--verbose", help="Show verbose output.", action="store_true"
```

Comment:
Might be useful to mention here that it overrides the silent option.

Reply:
Yeah, please do expand on this. Something like "Shows the output of all commands run, regardless of the silent argument."